tests/test-geometry.lua

```diff
@@ -257,6 +261,135 @@ local steps = {
     end
 }
 
-runner.run_steps(steps)
+print("Added tests for unmanaged client geometries.")
```

Comment:
i guess this should be a comment, not a print statement

Reply:
Done